### PR TITLE
[Merged by Bors] - feat: Add `Real.pi_nonneg = Real.pi_pos.le`

### PR DESCRIPTION
**Pull request by girving**
I think if `Real.exp_nonneg` exists as a separate lemma, `Real.pi_nonneg` is reasonable.  And it's useful as a separate lemma for Aesop purposes too.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean [hunk 1]
```diff
@@ -164,6 +164,9 @@ theorem pi_pos : 0 < π :=
   lt_of_lt_of_le (by norm_num) two_le_pi
 #align real.pi_pos Real.pi_pos
 
+theorem pi_nonneg : 0 ≤ π :=
+  pi_pos.le
+
 theorem pi_ne_zero : π ≠ 0 :=
   ne_of_gt pi_pos
```
Comment (adomani) [inline Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean:171]:
```suggestion
   pi_pos.ne'
```
Does it work?

Comment (girving) [reply Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean]: Oops, your suggested change had three spaces instead of one. :/

Comment (adomani) [reply Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean]: Sorry about that!  I was on mobile...

**Comment by adomani** [comment]
I am not sure where to draw the line on lemmas about "important" constants.  In particular, on the one hand, I understand that these lemmas can be helpful for certain tactics, on the other hand these lemmas also seem the ones that automation could make obsolete!

I'm pinging @urkud and @sgouezel who can make a more informed decision about this PR.

**Comment by sgouezel** [comment]
This one will show up so often that it's definitely worth having explicitely. Thanks!
bors r+

**Pull request closed without merging**